Makefile

```diff
@@ -28,7 +28,7 @@ JEKYLL_OPTS := -d '$(SITE_DESTDIR)' $(if $(SITE_BASEURL),-b '$(SITE_BASEURL)',)
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.2.1)
 # - use environment variables to overwrite this value (e.g export VERSION=0.2.1)
-VERSION := $(shell git describe --tags --dirty --always)
+VERSION := 0.4.0
```

Comment:
We actually don't need to change this here. When the release get's tagged the version will be exactly right. Moreover, it will be correct for any possible further staging versions on the release branch and point releases, too